src/mnode/mnode-pastel.cpp

```diff
@@ -1174,13 +1174,68 @@ CArtTradeTicket CArtTradeTicket::Create(std::string _sellTnxId, std::string _buy
     ticket.price = sellTicket->askedPrice;
 
     ticket.GenerateTimestamp();
-    
+
+    // In case it is nested it means that we have the artTnxId of the sell ticket
+    // available within the trade tickets.
+    // [0]: original registration ticket's txid
+    // [1]: copy number for a given art
+    std::vector<std::string> artRegTicket_TxId_Serial = CArtTradeTicket::GetArtRegTxIDAndSerialIfResoldNft(sellTicket->artTnxId);
+    if(artRegTicket_TxId_Serial[0].compare("") == 0)
```

Comment:
function returns all time vector with size 2. any case do we need checking here that list is not empty?

Reply:
he is comparing the string, not the size of vector

Reply:
I know. but this function returns either 2 empty strings or 2 not empty. I think no need to create a vector for this. function will look more clear if return bool. or can return std::tuple instead of vector. or structure.

Reply:
I better leave it like this. As wrote above:
Because purpose is a comparison, and if we use empty it could be misleading in terms of the logic if someone reads freedcamp task. Since both gives same result, no issue here.

Reply:
+ return value is feed into the RPC response. I'll leave it as is.